rules/import-style.js

```diff
@@ -172,7 +172,7 @@ const create = context => {
 	styles = new Map(
 		Object.entries(styles).map(
 			([moduleName, styles]) =>
-				[moduleName, new Map(Object.entries(styles))]
+				[moduleName, new Set(Object.entries(styles).filter(([_, isAllowed]) => isAllowed).map(([moduleName]) => moduleName))]
```

Comment:
```suggestion
				[moduleName, new Set(Object.entries(styles).filter(([, isAllowed]) => isAllowed).map(([moduleName]) => moduleName))]
```

Reply:
The variable name seems not right too?

```suggestion
				[moduleName, new Set(Object.entries(styles).filter(([, isAllowed]) => isAllowed).map(([importStyle]) => importStyle))]
```